app.py

```diff
@@ -33,6 +33,7 @@
 
 app = Flask(__name__)
 app.config.from_object(os.environ.get('APP_SETTINGS'))
+app.config['SECURITY_PASSWORD_SALT'] = '$2a$16$PnnIgfMwkOjGX4SkHqSOPO'
```

Comment:
Not too well versed in security, though I think that committing the salt to version control would defeat the purpose